Sources/HiPMobile/BusinessLayer/UserManagement/UserManager.cs

```diff
@@ -37,6 +38,7 @@ public class UserManager : IUserManager
                 Settings.Username = user.Username;
                 Settings.Password = user.Password;
                 Settings.AccessToken = user.Token.AccessToken;
+                await IoCManager.Resolve<IFeatureToggleRouter>().RefreshEnabledFeaturesAsync();
             }
 
             catch (Exception ex)
```

Comment:
This is not part of your task, but if have the time, could you add brackets to the following lines? Thanks!